### ISSUE TITLE
Can I get logits for each sequence I acqired from model.generate()?

### ISSUE THREAD

**Issue by RandolphShi** [huggingface/transformers#8794]
Hi, I’m currently stucked in getting logits from model.generate. I’m wondering if it is possible to get logits of each seqeucne returned by model.generate. (like logits for each token returned by model.logits)

**Comment by patrickvonplaten** [comment]
Sadly not at the moment... -> we could introduce such a feature soon though!
